docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,95 @@
+* *{filebeat}* - Install {filebeat-ref}/filebeat-installation-configuration.html[{filebeat}] version 8.x. Earlier {filebeat} versions are incompatible with ECS and will prevent indicator data from displaying in the Indicators table.
+--
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+Threat intelligence is a research function that helps analyze current and emerging threats, and recommends appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.
+
+An indicator, also referred to as an indicator of compromise (IoC), is a piece of information associated with a known threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
+
+[discrete]
+[[setup-indicators-page]]
+== Set up the Indicators page
+
+Install a threat intelligence integration to add indicators to the Indicators page.
+
+NOTE: If you've already installed a threat intelligence integration, you don't need to complete the following steps. Indicator data will automatically display on the Indicators page.
+
+. Choose one of the following:
+* From the {security-app} main menu, go to *Intelligence* -> *Add Integrations*.
+* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter by threat intelligence integrations.
+. Select a threat intelligence integration, then complete the integration's guided installation.
++
+NOTE: For more information about available fields, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.
+
+. Return to the Indicators page in {elastic-sec}. Refresh the page if indicator data isn't displaying.
+
+[discrete]
+[[troubleshoot-indicators-page]]
+=== Troubleshooting
+If indicator data is not appearing in the Indicators table after you installed a threat intelligence integration:
+
+* Verify that the index holding indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index holding indicator documents will differ based on the way you're collecting indicator data:
+** *{agent} integrations* - If you are collecting indicator data from {agent} threat intelligence integrations, indicator documents are stored in the `logs_ti*` index pattern.
+** *{filebeat} integrations* - If you are collecting indicator data from {filebeat} threat intelligence integrations, indicator documents are stored in the `filebeat-*` index pattern.
```

Comment:
Not sure about indices "holding" documents -- maybe "storing"?

Also, the bullets and lead-in are very wordy and repetitive, making readers do so much work to get simple information. This seems perfect for a table or definition list:

> Each indicator data collection method uses a different index:
> 
> - **Elastic Agent integrations**: `logs_ti*`
> - **Filebeat integrations**: `filebeat-*`
>

Reply:
Yeah, storing is a good alternative. It makes sense to say that indices _store_ documents.

Reply:
```suggestion
* Verify that the index storing indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index storing indicator documents will differ based on the way you're collecting indicator data:
** *{agent} integrations* - `logs_ti*` 
** *{filebeat} integrations* - `filebeat-*`
```